python/ray/autoscaler/aws/example-gpu-docker.yaml

```diff
@@ -18,7 +18,7 @@ initial_workers: 0
 # and opens all the necessary ports to support the Ray cluster.
 # Empty string means disabled.
 docker:
-    image: "tensorflow/tensorflow:latest-gpu"
+    image: "tensorflow/tensorflow:1.12.0-gpu"
```

Comment:
This is needed because the DL AMI v21 doesn't support the latest TF + CUDA driver pairing